src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
+      let userName = $('.username.input').val();
+
+      fetch('https://api.github.com/users/' + userName)
```

Comment:
Good practise is to put http requests in separate files with the same business logic. So we keep code simple and easy to maintain. I would recommend to create ProfileService which store http request for github user and method which updates profile. So every action on this profile will be on one place. App.js will be cleaner and prepared for more events which do not mess in class.